library/src/main/java/com/bumptech/glide/load/resource/gif/GifDrawable.java

```diff
@@ -390,6 +407,47 @@ public void setLoopCount(int loopCount) {
     }
   }
 
+  /**
+   * Register callback to listen to GifDrawable animation end event after specific loop count
+   * set by {@link GifDrawable#setLoopCount(int)}
```

Comment:
Worth noting that this will only be called if the Gif stops because it reaches the loop count. It doesn't look like you're notifying if someone calls stop() directly.